unity-client/Assets/Rendering/Utils/ShaderUtils.cs

```diff
@@ -0,0 +1,40 @@
+﻿using UnityEngine;
+
+namespace DCL.Helpers
+{
+    public static class ShaderUtils
+    {
+        public static readonly int _LightDir = Shader.PropertyToID("_LightDir");
```

Comment:
Just an observation here: Is it possible to remove the underscore prefix and name those fields in camel case?
I know this class already existed and was promoted to it own file and that fields are named to match shader's property name.
But it doesn't match any of our notations in this project, plus, even if it's not standardized by Microsoft, a really great number of C# devs reserve the underscore prefix to distinguish private fields

Reply:
I'd like this too XD

Reply:
Yeah, your arguments make sense @pbosio . Just let's discuss this as another issue and dedicate another PR to it 👍